File: 3522-find-the-power-of-k-size-subarrays-i/solution.java
class Solution {
    public int[] resultsArray(int[] nums, int k) {
        int n = nums.length;
        int[] ans = new int[n-k+1];
        for(int i = 0; i<=n-k; i++){
            int power = nums[i+k-1];
            for(int j = i+1; j<i+k; j++){
                if(nums[j] != nums[j-1]+1){
                    power = -1;
                    break;
                }
            }
            ans[i] = power;
        }
        return ans;
    }
}

File: 3522-find-the-power-of-k-size-subarrays-i/solution.py
class Solution:
    def resultsArray(self, nums: List[int], k: int) -> List[int]:
        n = len(nums)
        ans = [0] * (n-k+1)
        for i in range(n-k+1):
            power = nums[i+k-1]
            for j in range(i+1, i+k):
                if(nums[j] != nums[j-1]+1):
                    power = -1
                    break
            ans[i] = power
        return ans
